testsuite/dockerfiles/uyuni-master-testsuite/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM registry.opensuse.org/systemsmanagement/uyuni/master/docker/containers/uyuni-master-testsuite
```

Comment:
From where this container image comes, and why we decided to name it `uyuni-master-testsuite`, what exactly contains?
I think the name don't clearly describe what it does.

Reply:
It is the name used in the other profiles, so I didn't want to change it. Makes sense?

Reply:
Aha ok, I see now in the code... :/ 
Personally I think the name it was not very well choosen, but yeah, make sense to keep this aligned with others 🤷🏼‍♂️